final/problem-def-ana/problem-ana.tex

```diff
@@ -95,9 +96,9 @@ \subsubsection{Language specific additional commands}
 One possible solution is to allow for language specific configurations
 that are loaded with the language definition of that language. This
 can be done by extending the editor services discussed in
-\cref{ssec:editor-serv} with REPL commands: similar to menu actions,
-where one can define menu button ``Run'' to run a program, one may
-define a ``load-file'' command and bind it to an action.
+\cref{ssec:editor-serv} with REPL commands. Similar to menu actions,
+where one can define menu buttons such as ``Run'' to run a program, one may
+then even bind the ``load-file'' command to an action.
 
 \subsubsection{Redefining terms bound to names}
```

Comment:
This subsection can maybe stay, as it is not entirely irrelevant. It is just that the language designer has to implement this, and not us. Our job is to allow the language designer to configure this. So maybe it should be reworded to reflect this.